nlpsandboxclient/api_client.py

```diff
@@ -50,25 +55,32 @@ def rest_get(self, uri: str, endpoint: str = None,
     def rest_post(self, uri: str, body: str, endpoint: str = None):
         """Sends an HTTP POST request."""
         response = self._rest_call(
-            'post', uri, body, endpoint,
-            headers={'Content-Type': 'application/json'}
+            'post', uri, body, endpoint
         )
         return _return_rest_body(response)
 
     def rest_get_paginated(self, uri, limit=10, offset=0):
         """Get pagniated rest call"""
-        new_uri = utils._limit_and_offset(uri, limit=limit, offset=offset)
+        new_uri = core.utils._limit_and_offset(uri, limit=limit, offset=offset)
         while new_uri:
             page = self.rest_get(new_uri)
             new_uri = page['links']['next']
-            yield page
+            # Make sure to only return the list of resources
+            page.pop("limit")
```

Comment:
Consider one liner solution `[page.pop(key) for key in ["limit", "links", "offset"]] `

Reply:
I thought of using a for loop here, no need for list comprehension because we aren't storing those values.